listenbrainz_spark/year_in_music/listening_time.py

```diff
@@ -20,14 +21,14 @@ def get_listening_time(year):
 
 
 def _get_total_listening_time():
-    # get recording length from mb_metadata_cache, if listen is unmapped default to 3 minutes.
+    # get recording length from recording table, if listen is unmapped default to 3 minutes.
```

Comment:
Is this because we have direct access to the main servers now?

Reply:
Yes.

Reply:
And we already use this table elsewhere.